torch_glow/src/PyTorchModelLoader.cpp

```diff
@@ -45,7 +45,7 @@ inline llvm::Expected<float> to32Bit(double val) {
 inline llvm::Expected<int32_t> to32Bit(int64_t val) {
   RETURN_ERR_IF_NOT(val <= std::numeric_limits<int32_t>::max() ||
                         val >= std::numeric_limits<int32_t>::lowest(),
-                    glow::strFormat("Value %ld is out of limit.", val));
+                    glow::strFormat("Value %lld is out of limit.", val));
```

Comment:
Doesn't like this, int64_t is a long I guess

Reply:
I don't know much about this, but interestingly int64_t seems to be a long on CI and long long on macos. Maybe we should just static cast this to long long for this print?

Reply:
Here is the problem:
cmake as a part of python build requires %lld:
<<< 
PyTorchModelLoader.cpp:48:67: warning: format specifies type 'long' but the argument has type 'int64_t' (aka 'long long') [-Wformat]
                    glow::strFormat("Value %ld is out of limit.", val));
                                           ~~~                    ^~~
                                           %lld
>>>

Whereas GitHub build requires %ld, I will make change for GitHub compliance.

Reply:
If we cast it then we should be able to make both happy